### DISCUSSION TITLE
sony sw3 as standalone

### DISCUSSION THREAD

**Discussion by avi532** [Navid200/xDrip#115] [Dexcom]
i recently purchased a sony sw3 to be used as standalone via xdrip to dexcom (since i have seen many people say that it works with g6 as standalone). I had to sideload google services since it no longer supports this old style watch. and i got xdrip onto the watch (i dont see other apps or the playstore). i get readings on the phone and on the watch but when i do force collect it doesnt show on neither. has anyone had this issue. (what interesting is that on the wear os on the phone is connected to the watch via bluetooth, on the watch under bluetooth settings it doesnt show the phone at all or the dexcom even after scanning.
any thoughts would be appreciated

**Comment by miketosh** [comment]
There was a step I need to do. After force collect on watch, I had to enable location, and then restart collector in order to have the watch talk directly to the dex. Before I did that, it was just getting values from the phone.

**Comment by miketosh** [comment]
I don't remember how I enabled location, but to restart collector, you got the three dots on top left corner, then the circle of arrows icon, third along top row. 

Once all this if done, it takes a few cycles before it picks up. Watch the logs. Then go look at the android wear page on the phone, and via the "device running collection service" value. Should be watch name and serial number.

**Comment by TrevorLambert** [comment]
It's been a while since I used it, but I'm pretty sure the only way it works is if you select "only use wear".
